build-support/bin/native/bootstrap.sh

```diff
@@ -40,11 +43,18 @@ function calculate_current_hash() {
   # sensitive to the CWD, and the `--work-tree` option doesn't seem to resolve that.
   (
    cd ${REPO_ROOT}
-   git ls-files -c -o --exclude-standard "${NATIVE_ROOT}" | \
-      git hash-object -t blob --stdin-paths | fingerprint_data
+   git ls-files -c -o \
+     --exclude-standard "${NATIVE_ROOT}" \
```

Comment:
AFAIK, `--exclude-standard` doesn't take an argument, so you should only need to include it once.